src/ML-METATENSOR/pair_metatensor.h

```diff
@@ -0,0 +1,73 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+#ifdef PAIR_CLASS
+// clang-format off
+PairStyle(metatensor, PairMetatensor);
+// clang-format on
+#else
+
+#ifndef LMP_PAIR_METATENSOR_H
+#define LMP_PAIR_METATENSOR_H
+
+#include <memory>
+
+#include "pair.h"
+
+#include <metatensor/torch/atomistic.hpp>
```

Comment:
This include should not be in this header as it can create conflicts due to having to include all pair_<style>.h header files at the same time. The details of the implementation should be hidden from general LAMMPS. See comment below.